deploy/hotColdWalletInteraction.ts

```diff
@@ -0,0 +1,138 @@
+import { utils, Wallet, Provider, Contract, EIP712Signer, types, ECDSASmartAccount } from "zksync-ethers";
+import * as ethers from "ethers";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+// load env file
+import dotenv from "dotenv";
+import { util } from "chai";
+dotenv.config();
+
+/*
+  1. We modify the signature format by adding a prefix to indicate which wallet signed the transaction.
+  2. Interactions with zkSync are similar to Ethereum with some zkSync-specific modifications.
+  3. Ensure you handle errors and edge cases, such as insufficient balance or invalid signatures, as needed.
+
+
+*/
+
+// Put the address of your AA factory
+const AA_FACTORY_ADDRESS = "0xA0aE477D79AA21c4EE156cDfDA37dE0b5ac296B0";
+
+// load the values into .env file after deploying the FactoryAccount
+const DEPLOYED_ACCOUNT_OWNER_PRIVATE_KEY = process.env.DEPLOYED_ACCOUNT_OWNER_PRIVATE_KEY || "";
+const ETH_ADDRESS = process.env.ETH_ADDRESS || "0x000000000000000000000000000000000000800A";
+const ACCOUNT_ADDRESS = process.env.DEPLOYED_ACCOUNT_ADDRESS || "";
+
+export default async function (hre: HardhatRuntimeEnvironment) {
+  // @ts-ignore target zkSyncSepoliaTestnet in config file which can be testnet or local
+  const provider = new Provider(hre.config.networks.zkSyncSepoliaTestnet.url);
+
+  // Private key of the account used to deploy
+  const wallet = new Wallet(DEPLOYED_ACCOUNT_OWNER_PRIVATE_KEY).connect(provider);
+  const factoryArtifact = await hre.artifacts.readArtifact("AAFactory");
+  const aaFactory = new ethers.Contract(AA_FACTORY_ADDRESS, factoryArtifact.abi, wallet);
+
+
+  // Initialize hot and cold wallets
+  const hotWalletPrivateKey = Wallet.createRandom().privateKey;
+  const coldWalletPrivateKey = Wallet.createRandom().privateKey;
+  const hotWallet = new Wallet(hotWalletPrivateKey, provider);
+  const coldWallet = new Wallet(coldWalletPrivateKey, provider);
+
+  // For the simplicity of the tutorial, we will use zero hash as salt
+  const salt = ethers.ZeroHash;
+
+  // deploy account owned by one owner but with two wallets, hot and cold wallet
+  const tx = await aaFactory.deployAccount(salt, hotWallet, coldWallet);
```

Comment:
You need to pass some `maxValue` there
```suggestion
  const tx = await aaFactory.deployAccount(salt, hotWallet, coldWallet, 0);
```